### DISCUSSION TITLE
Use picture tag in commonmark

### DISCUSSION THREAD

**Discussion by karaka200** [thephpleague/commonmark#925] [Q&A] [answered]
Hi
How can I use picture tag in commonmark?
```
<picture>
  <source type="image/webp" srcset="flower.webp">
  <source type="image/jpeg" srcset="flower.jpg">
  <img src="flower.jpg" alt="">
</picture>
```

**Comment by colinodell** [comment, marked as answer]
Hi @karaka200!

[The CommonMark spec](https://spec.commonmark.org/0.30/) that we follow unfortunately doesn't provide any kind of Markdown syntax that can be used to generate `<picture>` tags, so you'll need to provide the [raw HTML](https://spec.commonmark.org/0.30/#raw-html) as part of your Markdown to use this.

**Reply by karaka200** [reply]
The problem is that `WEBP` is not supported in some browsers and I am stuck between using only `WEBP` or `JPG`.
